### ISSUE TITLE
Typography as components

### ISSUE THREAD

**Issue by LBF38** [huntabyte/shadcn-svelte#1123]
### Describe the feature

Hi there, 
It could be nice to have typography as components for easier text styling/structuring when building pages.

Here are the references for the issues/PRs in the `shadcn/ui` repo : 
- https://github.com/shadcn-ui/ui/issues/315
- https://github.com/shadcn-ui/ui/pull/363

Thanks for considering it !

**Comment by huntabyte** [comment]
Hey there! If these are added to the original then we'll add them here, but not until then! Going to close this for now but will reopen should that time come.
